scripts/templates/index.d.ts.tpl

```diff
@@ -8,9 +8,9 @@ declare namespace Github {
   type json = any
   type date = string
 
-  export interface AnyResponse {
+  export interface AnyResponse<T> {
```

Comment:
Would it make sense now to rename it to `Response<T>`? An alias `AnyResponse = Response<any>` could be kept too if needed for compatibility.

Reply:
You can also do `Response<T = any>`